tests/test_main.cpp

```diff
@@ -121,6 +121,7 @@
 #include "tests/scene/test_sprite_frames.h"
 #include "tests/scene/test_theme.h"
 #include "tests/scene/test_timer.h"
+#include "tests/scene/test_tree.h"
```

Comment:
Should be moved down to the `#ifndef ADVANCED_GUI_DISABLED` part as it is advanced UI

Reply:
Moved to advanced gui section